src/Redux/Books/Books.js

```diff
@@ -0,0 +1,5 @@
+const Books = () => (
+  <h1>Books components</h1>
+);
+
```

Comment:
- [ ] It would be better if the book component Takes the book data as a prop and renders the title and the author. kindly add this to the book component. 

- [ ] It would be better if you has a button for removing books. The button should not remove the books yet, just a placeholder for now

- [ ] Please add the form to add a book. 

![Capture](https://user-images.githubusercontent.com/64173189/136294249-43cd8125-1939-44df-bc2c-5cd91981e5e3.JPG)


👍 🛠️